rules/properties-order/README.md

```diff
@@ -594,6 +595,67 @@ a {
 }
 ```
 
+### `emptyLineBeforeUnspecified: "always"|"never"`
+
+These options only apply if you've defined your own array of properties.
```

Comment:
This paragraph is no necessary, because for `properties-order` array of properties will always be specified.